openlibrary/templates/lib/nav_head.html

```diff
@@ -3,7 +3,7 @@
 <header id="header-bar" class="header-bar">
   <div class="hamburger-component">
 
-    <label role="button" for="toggle-hamburger" class="hamburger-button">
+    <label for="toggle-hamburger" class="hamburger-button">
```

Comment:
The role is there for accessibility. Am curious what the error was here. I think the role needs to be on the input[typecheckbox] on L10 instead.

Reply:
```
Error: Attribute role not allowed on element label at this point.

From line 107, column 5; to line 107, column 73

nt">↩↩    <label role="button" for="toggle-hamburger" class="hamburger-button">↩     

Attributes for element label:
    Global attributes
    for — Associate the label with form control
```

Reply:
Yep so looks like this was meant for the input[type=checkbox]. Nice catch!